oracle/oracle.go

```diff
@@ -639,14 +636,18 @@ func (o *oracleOps) waitVolumeAttachmentStatus(volumeAttachmentID *string, desir
 			return nil, true, err
 		}
 		if getVolAttachmentResp.GetLifecycleState() == desiredStatus {
-			return nil, false, nil
+			return getVolAttachmentResp.GetDevice(), false, nil
 		}
 
 		logrus.Debugf("volume [%s] is still in [%s] state", *getVolAttachmentResp.GetVolumeId(), getVolAttachmentResp.GetLifecycleState())
 		return nil, true, fmt.Errorf("volume [%s] is still in [%s] state", *getVolAttachmentResp.GetVolumeId(), getVolAttachmentResp.GetLifecycleState())
 	}
-	_, err := task.DoRetryWithTimeout(f, cloudops.ProviderOpsTimeout, cloudops.ProviderOpsRetryInterval)
-	return err
+	devicePathRaw, err := task.DoRetryWithTimeout(f, cloudops.ProviderOpsTimeout, cloudops.ProviderOpsRetryInterval)
```

Comment:
How did you test that the value of devicePathRaw was in the format you expect? 

In general for all PRs, let's try to add testing details in a "Testing" section under "Notes".